code/modules/antagonists/cult/cult_items.dm

```diff
@@ -931,7 +931,7 @@
 		if(istype(hitby, /obj/item/projectile))
 			var/obj/item/projectile/P = hitby
 			if(P.damage_type == BRUTE || P.damage_type == BURN)
```

Comment:
I'd say remove the burn check and maybe buff the reflect chance a bit so it still punishes laser/disabler spam to make up for the fact you can break it with WT 550s and the like